doc/api/index.rst

```diff
@@ -114,10 +114,33 @@ Alphabetical list of modules:
 Usage patterns
 --------------
 
-Below we describe several common approaches to plotting with Matplotlib.
+Below we describe several common approaches to plotting with Matplotlib.  See
+:ref:`api_interfaces` for an explanation of the trade off of the supported user
+APIs.
 
-The pyplot API
-^^^^^^^^^^^^^^
+
+The Explicit API
+^^^^^^^^^^^^^^^^
+
+At its core, Matplotlib is an object-oriented library. We recommend directly
+working with the objects, if you need more control and customization of your
+plots.
+
+In many cases you will create a `.Figure` and one or more
+`~matplotlib.axes.Axes` using `.pyplot.subplots` and from then on only work
+on these objects. However, it's also possible to create `.Figure`\ s
+explicitly (e.g. when including them in GUI applications).
+
+Further reading:
+
+- `matplotlib.axes.Axes` and `matplotlib.figure.Figure` for an overview of
+   plotting functions.
+- Most of the :ref:`examples <examples-index>` use the object-oriented approach
+   (except for the pyplot section)
```

Comment:
This renders a bit weird with bold on the first line and standard text on the second for these bullets.